src/main/java/uk/gov/hmcts/reform/sscs/services/ccd/UpdateCcdRepresentative.java

```diff
@@ -31,6 +40,23 @@ static boolean updateCcdRepresentative(SscsCaseData gapsCaseData, SscsCaseData e
         return repUpdated;
     }
 
+    private static String checkValidUkMobileNumberElseReturnExisting(Representative rep,
+                                                                     Representative existingRepresentative,
+                                                                     String caseReference) {
+        String validMobileNumber = null;
+        boolean isValidUkMobile = UkMobile.check(rep.getContact().getMobile());
+        if (isValidUkMobile) {
+            validMobileNumber = rep.getContact().getMobile();
+        } else if (existingRepresentative != null && existingRepresentative.getContact() != null) {
+            validMobileNumber = existingRepresentative.getContact().getMobile();
+        }
+
+        if (!isValidUkMobile) {
+            LOG.error("Invalid Uk mobile no: {} for the case: {}", caseReference, rep.getContact().getMobile());
```

Comment:
this is triggering an alert. It's better to log it as  INFO. It's not an error, just something we want to log to track it.

Reply:
Ok